public/css/404.css

```diff
@@ -0,0 +1,114 @@
+* {
+  -webkit-box-sizing: border-box;
```

Comment:
Can we indent by 4 spaces instead of 2, please?  I'm not a CSS expert, so if there is a convention here that I am unaware of, please let me know.